example/app-model/src/main/java/com/mastercard/test/flow/example/app/model/Direct.java

```diff
@@ -96,19 +96,16 @@ public Direct() {
 						.motivation( m -> m + ", in Switzerland" ) )
 				.update( UI,
 						rq( "o", "oOoOo" ),
-						rs( "O", 4,
-								"o", 6 ) )
+						rs( "o", 10 ) )
 				.update( CORE,
 						rq( "text", "HelloOoOo woOoOorld!" ),
-						rs( "result.o", 6,
-								"result.O", 4 ) )
+						rs( "result.o", 10 ) )
 				.update( HISTOGRAM,
 						rq( "o", "oOoOo" ),
-						rs( "O", 4,
-								"o", 6 ) ) );
+						rs( "o", 10 ) ) );
 
 		List<Object> deletes = valueFill( DELETE,
-				" ", "!", "H", "d", "l", "r", "w" );
+				" ", "!", "h", "d", "l", "r", "w" );
```

Comment:
More test-data changes to make our existing test reflect the new behaviour of the system.